homeassistant/components/ssdp/__init__.py

```diff
@@ -33,12 +34,12 @@
 async def async_setup(hass, config):
     """Set up the SSDP integration."""
 
-    async def initialize():
+    async def initialize(_):
         scanner = Scanner(hass)
         await scanner.async_scan(None)
         async_track_time_interval(hass, scanner.async_scan, SCAN_INTERVAL)
 
-    hass.loop.create_task(initialize())
+    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, initialize)
```

Comment:
I think we should wait till the STARTED event? START task listeners are still awaited.

Reply:
Probably should change discovery over as well.